run.sh

```diff
@@ -1,17 +1,22 @@
 #!/bin/sh
 
-set -e
+# start in right dir
+cd /ircd
 
-if [ ! -f ./ircd.yaml ]; then
-  cp oragono.yaml ircd.yaml
+# make config file
+if [ ! -f "/ircd/ircd.yaml" ]; then
+    awk '{gsub(/path: languages/,"path: /ircd-bin/languages")}1' /ircd-bin/oragono.yaml > /tmp/ircd.yaml
+    mv /tmp/ircd.yaml /ircd/ircd.yaml
 fi
 
-if [ ! -f ircd.db ]; then
-  ./oragono initdb
+# make db and certs
+if [ ! -f "/ircd/ircd.db" ]; then
+    /ircd-bin/oragono initdb
 fi
 
-if [ ! -f tls.crt ]; then
-  ./oragono mkcerts
+if [ ! -f "/ircd/tls.key" ]; then
+    /ircd-bin/oragono mkcerts
 fi
 
-exec ./oragono run
+# run!
+/ircd-bin/oragono run
```

Comment:
Probably want to use `exec` here so the signals get passed on. Also not sure if we should be specifying an entrypoint in the dockerfile rather than a command? https://hynek.me/articles/docker-signals/ is usually my go-to guide for figuring this stuff out but it's not totally clear on CMD vs ENTRYPOINT.
```suggestion
exec /ircd-bin/oragono run
```

Reply:
Ah, iiinteresting. Thanks, I'll take a look at this to see how it all works!